app/restore_to_file.go

```diff
@@ -21,6 +24,8 @@ const (
 	BackupFilePath        = "backup.img"
 	BackupFileConverted   = "backup.img.converted"
 	BackingFileCopy       = "backing.img.cp"
+
+	Retry = 5
```

Comment:
Still the same name...

Also, it should be in minutes. e.g. `ProgressBasedTimeout = 5 * time.Minute`

5 seconds is too soon. TCP retry is longer than that.

Reply:
As restore_to_file is a blocking operation, I thought of keeping it smaller. Do you suggest to keep it 5 minutes here as well?
The only downside would be to wait for 5 minutes even for shorter restores.